en/sdk/cocos-services.md

```diff
@@ -1,11 +1,9 @@
 # Cocos Services
 
-Cocos Creator from **v2.0.7** began to add a **Service** panel to the **Menu bar -> Panel**, through the **Service** panel, users can quickly integrate services for the game.
+Starting with **Cocos Creator v2.0.7**, users can quickly integrate services using the **Service** panel. To get to the **Service** panel by clicking **Menu bar -> Panel -> Service**. The **Service** panel currently supports integrating **AnySDK**. Please refer to the [AnySDK documentation](anysdk.md) for details.
```

Comment:
> Since you have supplemented the **Cocos Analytics** section of the EN document, so you need to add the Analytics service here. Please refer the ZH document https://docs.cocos.com/creator/manual/zh/sdk/cocos-services.html

I don't understand this text.

Reply:
@xunyi0 I dont understand the text you wrote above.

Reply:
@slackmoehrle This text means that you need to update the [Cocos Servers](https://docs.cocos.com/creator/manual/en/sdk/cocos-services.html) section of the English doc to supplement the content of Cocos Analytics. Please refer to the Chinese doc <https://docs.cocos.com/creator/manual/zh/sdk/cocos-services.html> for details.

Reply:
Thanks. I will pull the latest and go through these next edits.